src/pages/resultsView/survival/SurvivalUtil.tsx

```diff
@@ -370,3 +401,20 @@ export function getSurvivalChartDataByAlteredStatus(
         patientToAnalysisGroups,
     };
 }
+
+export function generateSurvivalPlotTitleFromDisplayName(title: string) {
+    return title.replace(/status|survival/gi, '');
```

Comment:
just want to make sure you know that the "g" modifier makes this global.  so it, will replace all instances of that word in the title.    it seems unlikely that there would be more than one.    you don't have to change this, i just wanted to make sure

Reply:
yes I know that, thanks